llama-index-integrations/llms/llama-index-llms-vertex/llama_index/llms/vertex/gemini_utils.py

```diff
@@ -46,14 +46,22 @@ def _convert_gemini_part_to_prompt(part: Union[str, Dict]) -> Part:
             raise ValueError("Only text and image_url types are supported!")
         return Part.from_image(image)
 
-    raw_content = message.content
+    if message.content == "Function Calling":
```

Comment:
They really call it `Function Calling` with no spaces? lol

Reply:
@logan-markewich Hi Logan, thanks merging my PR!

The message content is actually set manually here to `Function Calling`
https://github.com/run-llama/llama_index/blob/96a196a9d0bed9ea8a3b9f9caac18639faaf0249/llama-index-core/llama_index/core/utilities/gemini_utils.py#L61-L62


Let me know if I should open another PR to clean this up